### PR TITLE
libvirt: enable not found in qemu line

### PR DESCRIPTION
**Pull request by dzhengfy**
This is to enable to verify some messages do not exist in the qemu command line.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

File: virttest/utils_test/libvirt.py [hunk 1]
```diff
@@ -3881,15 +3886,19 @@ def check_qemu_cmd_line(content, err_ignore=False,
...
-            raise exceptions.TestFail("Expected '%s' was NOT found in "
-                                      "qemu command line" % content)
-    return True
+            raise exceptions.TestFail("Expecting '%s' does%s "
+                                      "exist in qemu command line, "
+                                      "but %sfound" % (content,
+                                                       '' if expect_exist else ' not',
+                                                       ' not' if expect_exist else ''))
+    else:
+        LOG.info("Qemu command line check for '%s': PASS" % content)
```
Comment (Yingshun) [inline virttest/utils_test/libvirt.py:3900]:
```suggestion
        LOG.info("Qemu command line check for '%s': PASS", content)
```

**Review by Yingshun — COMMENTED** [review]
Others LGTM

**Review by chunfuwen — APPROVED** [review]
lgtm

File: virttest/utils_test/libvirt.py (hunk 1 above)
Comment (dzhengfy) [reply virttest/utils_test/libvirt.py]: Updated, thanks.

**Review by Yingshun — APPROVED** [review]

**Pull request merged**